views/breadDetails.jsx

```diff
@@ -0,0 +1,8 @@
+const React = require('react')
```

Comment:
This file will likely conflict make sure to accept Jeremy's changes since they were more polished

Reply:
accepted jeremy's changes